cli/cli_test.go

```diff
@@ -86,24 +84,43 @@ func TestRunDaemon(t *testing.T) {
 			signal: syscall.SIGUSR2,
 		},
 	}
+	config := conf.MenderConfig{
+		MenderConfigFromFile: conf.MenderConfigFromFile{
+			Servers: []client.MenderServer{{}},
+		},
+	}
+	pieces := app.MenderPieces{
+		Store: store.NewMemStore(),
+		DualRootfsDevice: installer.NewDualRootfsDevice(
+			nil, nil, installer.DualRootfsDeviceConfig{}),
+	}
+
+	pieces.AuthMgr = app.NewAuthManager(app.AuthManagerConfig{
+		AuthDataStore: pieces.Store,
+		KeyStore:      store.NewKeystore(pieces.Store, conf.DefaultKeyFile),
+		IdentitySource: &dev.IdentityDataRunner{
+			Cmdr: stest.NewTestOSCalls("mac=foobar", 0),
+		},
+	})
 
 	for name, test := range tests {
-		mender := newDefaultTestMender()
```

Comment:
Since you have added some repeated code above, could the `newDefaultTestMender()` function be moved to `github.com/mendersoftware/mender/tests`? That would make it usable from both places, and you wouldn't need this extra bit.

Reply:
I tried, but imposes an import cycle that's not possible to break (due to the dependency on the app package).